### PR TITLE
refact: rename EditableDataframe into EditableDataFrame

### PR DESCRIPTION
**Pull request by FabienArcellier**
image

### CODE REVIEW

File: src/ui/src/components/core/content/CoreDataframe.vue [hunk 1]
```diff
@@ -139,23 +139,30 @@ const description = "A component to display Pandas DataFrames.";
 
 const dataFrameUpdateHandlerStub = `
 # Subscribe this event handler to the \`wf-dataframe-update\` event
+#
+# more on : https://dev.writer.com/framework/dataframe
 def on_record_change(state, payload):
-    state['mydf'].record_update(payload)`;
+    state['mydf'].record_update(payload) # should contains an EditableDataFrame instance`;
```
Comment (ramedina86) [inline src/ui/src/components/core/content/CoreDataframe.vue:145]:
```suggestion
    state['mydf'].record_update(payload) # should contain an EditableDataFrame instance`;
```

File: src/ui/src/components/core/content/CoreDataframe.vue [hunk 2]
```diff
@@ -139,23 +139,30 @@ const description = "A component to display Pandas DataFrames.";
...
 
 const dataFrameAddHandlerStub = `
 # Subscribe this event handler to the \`wf-dataframe-add\` event
+#
+# more on : https://dev.writer.com/framework/dataframe
 def on_record_add(state, payload):
 	payload['record']['sales'] = 0 # default value inside the dataframe
-	state['mydf'].record_add(payload)`;
+	state['mydf'].record_add(payload) # should contains an EditableDataFrame instance`;
```
Comment (ramedina86) [inline src/ui/src/components/core/content/CoreDataframe.vue:153]:
```suggestion
	state['mydf'].record_add(payload) # should contain an EditableDataFrame instance`;
```

File: src/ui/src/components/core/content/CoreDataframe.vue [hunk 3]
```diff
@@ -139,23 +139,30 @@ const description = "A component to display Pandas DataFrames.";
...
 const dataFrameActionHandlerStub = `
 # Subscribe this event handler to the \`wf-dataframe-action\` event
+#
+# more on : https://dev.writer.com/framework/dataframe
 def on_record_action(state, payload):
+	# state['mydf'] should contains an EditableDataFrame instance
 	record_index = payload['record_index']
 	if payload['action'] == 'remove':
-		state['mydf'].record_remove(payload)
+		state['mydf'].record_remove(payload) # should contains an EditableDataFrame instance
```
Comment (ramedina86) [inline src/ui/src/components/core/content/CoreDataframe.vue:163]:
```suggestion
		state['mydf'].record_remove(payload) # should contain an EditableDataFrame instance
```

**Review by madeindjs — COMMENTED** [review]
I also found this place which mention `EditableDataframe`. Even though it's a test, it could be good to update

https://github.com/writer/writer-framework/blob/7b848a0a8e331dea3ec825232b525c6c9f387a4b/tests/backend/test_core.py#L1300

And maybe these classes names could be updated :thinking: 

https://github.com/writer/writer-framework/blob/00b964f43f5d22ea818573d94025e73b9fe17470/src/writer/ss_types.py#L191-L199

**Comment by ramedina86** [comment]
I think that goes beyond what we established as a goal (changing what the user sees). Though it's a potential source of confusion/bugs later on, yes. @madeindjs

opinions @FabienArcellier ?

**Comment by ramedina86** [comment]
I'm ok with changing everything just not `wf-dataframe-update` to avoid breaking apps

**Comment by FabienArcellier** [comment]
> I think that goes beyond what we established as a goal (changing what the user sees). Though it's a potential source of confusion/bugs later on, yes. @madeindjs
> 
> opinions @FabienArcellier ?

I have done the change

**Review by madeindjs — APPROVED** [review]

**Pull request merged**